scripts/preconversion_assessment_script.py

```diff
@@ -217,28 +219,60 @@ def run_subprocess(cmd, print_cmd=True, env=None):
     return output, process.returncode
 
 
+def _get_last_yum_transaction_id(pkg_name):
+    output, return_code = run_subprocess(["/usr/bin/yum", "history", "list", pkg_name])
+    if return_code:
+        # NOTE: There is only print because list will exit with 1 when no such transaction exist
+        print(
+            "Listing yum transaction history for '%s' failed with exist status '%s' and output '%s'"
+            % (pkg_name, return_code, output),
+            "\nThis may cause clean up function to not remove '%s' after Task run."
+            % pkg_name,
+        )
+        return None
+
+    pattern = re.compile(r"^(\s+)?(\d+)", re.MULTILINE)
```

Comment:
Small nitpick here, but we can change it later: We could move this to outside the function, and turn this into a global constant.

Reply:
what do you mean? the pattern as a constant? :thinking: hmm I mean why not, but I don't see a reason to do it, it's not useful to any other part of script, it's specific to the function purpose.